### PR TITLE
Issue 3457/assign all datapoints ui

### PR DESCRIPTION
**Pull request by marvinkome**
#### Before the PR (what is the issue or what needed to be done)
Add the Assign all data points to device feature

#### The solution
Add the UI for this feature and the functionality for it

#### Screenshots (if appropriate)
image

#### Reviewer Checklist
* [ ] Added an explanation about the work done
* [ ] Connected the PR and the issue on Zenhub
* [ ] Added a test plan to the issue
* [ ] Updated the copyright header (when relevant)
* [ ] Formatted the code
* [ ] Added a documentation (if relevant)
* [ ] Added some unit tests (if relevant)

### CODE REVIEW

File: Dashboard/app/js/lib/components/devices/AssignmentsEditView/screens/AssignDatapoints/index.jsx [hunk 1]
```diff
@@ -73,15 +66,47 @@ export default class AssignDatapoints extends React.Component {
...
+              </>
+            )}
+          </p>
+        </div>
 
-            <button onClick={() => this.changeTab('SEARCH_DATAPOINTS')} type="button">
-              {strings.assignDatapointByNameOrId}
-            </button>
-          </div>
+        <Dropdown disabled={data.allDataPointsAssigned} title="Assign Datapoints">
```
Comment (muloem) [inline Dashboard/app/js/lib/components/devices/AssignmentsEditView/screens/AssignDatapoints/index.jsx]: Should the title also be added as a string?

Comment (marvinkome) [reply Dashboard/app/js/lib/components/devices/AssignmentsEditView/screens/AssignDatapoints/index.jsx]: Do you mean translation?

Comment (muloem) [reply Dashboard/app/js/lib/components/devices/AssignmentsEditView/screens/AssignDatapoints/index.jsx]: Yes

Comment (marvinkome) [reply Dashboard/app/js/lib/components/devices/AssignmentsEditView/screens/AssignDatapoints/index.jsx]: Yes it should. Updating now

**Pull request merged**